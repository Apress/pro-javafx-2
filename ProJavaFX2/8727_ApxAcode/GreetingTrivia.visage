/*
 * @test
 * @run
 */

var name = "Weiqi";
var i = 1024;
var j = 9765625.0;
println(##[greeting]"Hello, {name}.");
println(##[trivia]"Do you know that "
  "{i} * {j} = {%,.1f i * j}?");
println(##"1024 is the 10th power of 2, "
  "and 9765625 is the 10th power of 5.");
