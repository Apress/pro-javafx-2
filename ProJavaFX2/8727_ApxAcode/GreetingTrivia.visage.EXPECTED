Hello, Weiqi.
Do you know that 1024 * 9765625.0 = 10,000,000,000.0?
1024 is the 10th power of 2, and 9765625 is the 10th power of 5.
